client/app/hosting/configuration/CONFIGURATION.html

```diff
@@ -0,0 +1,49 @@
+<div class="container-fluid px-0" data-ng-controller="HostingTabConfigurationCtrl as ctrl">
+    <div class="row">
+        <div class="col-md-9">
+            <div data-ovh-alert="{{ alerts.main }}"></div>
+
+            <oui-spinner data-size="l" data-ng-if="ctrl.loading.init"></oui-spinner>
+            <div data-ng-if="!ctrl.loading.init && ctrl.hasResult">
+                <div data-ng-include="ctrl.templates.runtimeList"></div>
+                <br><br>
+
+                <div data-ng-include="ctrl.templates.envvarList"></div>
```

Comment:
Two datagrids on the same page is definitely hard to read.
There are two paginations, two search bars... It clutters the interface.
We should consider putting the envvar feature on a different page, with a back button like the database backup page (Databases > Triple dot menu > Restore a backup) for example.

Reply:
It was a decision made from UX during a meeting two months ago.
I am not able to help with this part and I would recommand you to forward this fact to someone in UX team :)

Reply:
The old decisions taken two months ago are void. Right now it completely contradicts the new CX rules. But don't worry, I'll be there to help you make this change :)